internal/rbd/rbd_util.go

```diff
@@ -81,6 +81,35 @@ const (
 
 	// clusterNameKey cluster Key, set on RBD image.
 	clusterNameKey = "csi.ceph.com/cluster/name"
+
```

Comment:
`rbd_util.go` is really large already, new features like QoS benefit from being placed as much as possible in a new `qos.go` file. Please move everything there, thanks!

Reply:
done